tools/osbuild-image-test

```diff
@@ -413,6 +418,12 @@ def main():
         type=os.path.abspath,
         default="/var/lib/osbuild/out",
         help="directory where result objects are stored")
+    parser.add_argument(
+        "--imi-output",
```

Comment:
`imi`?  Is that short for `image-info`?
It's a big unclear.  I know `--image-info-output` is a bit long, but at least it's clear.